tests/conftest.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+import pytest
+
+
+@pytest.fixture
+def tmpdir():
```

Comment:
pytest ships with this I think https://docs.pytest.org/en/latest/tmpdir.html or was there a reason to reuse the existing tmpdir?